src/backend/startup.js

```diff
@@ -27,6 +27,19 @@ export async function checkRoomNameExists(roomName) {
     return false;
 }
 
+export async function checkPlayerNameExists(roomName, playerName) {
+    console.log("Checking Player Name...");
+    const currentPlayers = await firestore.collection(root).doc(roomName).collection("players").get();
+    for (let i=0; i<currentPlayers.docs.length; i++){
+        console.log(currentPlayers.docs[i].data().name);
+        if (currentPlayers.docs[i].data().name === playerName) {
```

Comment:
Technically 2 people can have the same name. Do we want to enforce unique names? I guess we need to so that people can perform actions on another person without confusion